packages/shared/styles/components/SfCharacteristic.scss

```
@@ -0,0 +1,25 @@
+@import '../variables';
+
+$characteristic-margin-right: 20px;
```

must use `!default` for scss vars:

```suggestion
$characteristic-margin-right: 20px !default;
```